README.md

```diff
@@ -18,18 +18,18 @@ file in the repository. The library can be implemented as a standalone tool or
 an importable module in your project. In both cases, the results can be written
 to an output file.</br>
 
-## Getting Started
+## Getting Started ##
 
 `findcdn` requires **Python 3.7+**. Python 2 is not supported. </br> `findcdn`
```

Comment:
Why a `</br>` here instead of just two returns to start a new paragraph?

Reply:
I'm not sure why `</br>` were used, but they have been removed in  9685a34.